### ISSUE TITLE
Feature Request: mysql_procedure resource

### ISSUE THREAD

**Issue by wobo-mattmencel** [petoju/terraform-provider-mysql#59]
Any chance a `mysql_procedure` resource could be added to this provider?

I've tested out doing this with a `local-exec`, but that method risks exposing passwords in the logs or in the STATE which I'm trying to avoid.

### Affected Resource(s)
Please list the resources as a list, for example:
- mysql_procedure

### References
There is a postgres provider that implements a similar functionality.
https://registry.terraform.io/providers/cyrilgdn/postgresql/latest/docs/resources/postgresql_function

**Comment by petoju** [comment]
There is a chance - if anyone sends out a PR of a reasonable quality, I'll merge it. I currently don't need it, so there is no plan to implement it from my side.

**Comment by Flasheh** [comment]
There's an undocumented `mysql_sql` resource which was created for this purpose. I've (ab)used it in the past to run some SQL statements though not with this fork of the provider.

Example usage can be found here: https://github.com/winebarrel/terraform-provider-mysql/pull/22

**Comment by sirantd** [comment]
> There is a chance - if anyone sends out a PR of a reasonable quality, I'll merge it. I currently don't need it, so there is no plan to implement it from my side.

@petoju Is it good enough https://github.com/petoju/terraform-provider-mysql/pull/292 ? I'd like to use it on my project if it's merged.

**Comment by petoju** [comment]
@sirantd  yes, I left some comments there - it looks like we'll be ready to merge it soon.
